src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1126,6 +1126,9 @@ ModuleVersion : Version of module to import. If used, ModuleName must represent
   <data name="ForEachTypeConversionFailed" xml:space="preserve">
     <value>Unable to convert input to the target type [{0}] passed to the ForEach() operator. Please check the specified type and try running your script again.</value>
   </data>
+  <data name="ForEachNotSupportCleanBlock" xml:space="preserve">
+    <value>Script block with a 'clean' block is not supported.</value>
+  </data>
```

Comment:
I think we need more clear message if it is about ForEach.

Reply:
Good point. Message updated to bee 
```suggestion
  <data name="ForEachNotSupportCleanBlock" xml:space="preserve">
    <value>Script block with a 'clean' block is not supported by the 'ForEach' method.</value>
  </data>
```